### ISSUE TITLE
Gmail.js not loading fast enough

### ISSUE THREAD

**Issue by nmyerson** [KartikTalwar/gmail.js#56]
I'm attempting to write a chrome extension using Gmail.js. Thank you!

I use the sample code:

var gmsrc = document.createElement('script');
    gmsrc.src = "https://xmailchrome.appspot.com/gmail.js";
    document.getElementsByTagName('body')[0].appendChild(gmsrc);
    console.log("Scripts loaded, Start playing ...");

But when I start playing it says: Uncaught reference error: Gmail is not defined. 

In the chrome console, after the script is ran... Gmail is defined. Just not when the code below 'start playing'  is ran. 

Basically I just want to use the Gmail.js onload. 

Thanks!

**Comment by josteink** [comment]
If you use NodeJS style require() and script-bundling through browserify load time is instant, which hopefully be good enough :)
